### PR TITLE
Page headers in the dashboard do not look the same.

### LINKED ISSUE #664: Page headers in the dashboard do not look the same.
The headers in the Library, New and Existing Analyses(1761), and Plugins, Pipelines and Compute pages look the same as below:
chris1

Meanwhile, the header in the PACS Query/Retrieve page is different.
chris2

And the DICOM Viewer page doesn't have a header at all.
chris5

I am an Outreachy applicant, and I would like to work on giving all the pages the same style and if possible make the page header a reusable component.

Please assign this to me. @jennydaman @mairin

### PR DESCRIPTION
**Pull request by Timonwa**
The PACS Query/Retrieve page header and the DICOM Viewer page header now correspond to the remaining page headers.

closes #664.

Before:
chris2
chris5

After:
chris12
FireShot Capture 045 - DICOM Viewer - localhost

### CODE REVIEW

**Comment by Timonwa** [comment]
@jennydaman @mairin @PintoGideon please check.

**Comment by Timonwa** [comment]
@mightyFZeus, please ensure that you do not have errors in your console log and that everything works fine before creating a pull request. The code you contributed to #657 had two errors;
- you forgot to close a curly bracket in the `createfeed.scss` file
- you didn't make the `date` value conditional in the `Study Date Picker` in the `QueryBuilder.tsx` file.

This was merged into the Master branch, causing errors for other people. I have fixed it in my pull request. Once this gets merged, the errors won't be there anymore.

**Comment by mightyFZeus** [comment]
@Timonwa Thanks for the fix, my bad

**Comment by Timonwa** [comment]
@jennydaman @mairin @PintoGideon also, I want to suggest that we create a `development ` or  `dev` branch where all current pull requests can be merged into, so it doesn't break the master branch and cause the main website to be broken also. We can also host the dev branch on vercel or netlify.  

Benefits of having and hosting a dev branch:
- company web app will not crash because contributed code has to be working well on the dev branch before being merged into the master branch
- if harmful code is merged into the dev branch, we can always fix it before it gets to the production branch, ie master branch
- when you deploy, and host on vercel or netlify, both of them feature automatic build of the app, 
     - so if there is any error causing the app not to build, it will notify us
     - it also runs its actions on every pull request made 
- vercel and netlify also create preview links for each pull request so we can view the changes made on the live app aside from just relying on the screenshots uploaded

**Comment by mightyFZeus** [comment]
@PintoGideon  @jennydaman

**Comment by jennydaman** [comment]
@Timonwa we actually did this before, take a look at what branches we have in this repository: https://github.com/FNNDSC/ChRIS_ui/branches

In previous runs of Outreachy, we asked contributors to target PRs to `outreachy/*` development branches. It led to a hell of merge conflicts because we were not strict about _gitiquette_. In the end we had to use cherry-picks and also leave out some of the contributions.

Currently, my intention is to be more strict about merging PRs and to have them target master. Hopefully this means we will be able to see more of people's work on our live deployment(s). If this gets out of hand, then we should think about having applicants target a new `outreachy/winter2022` branch.

p.s. **Unmerged PRs will still count towards your final Outreachy application.** Or more accurately, the Outreachy intern selection will be holistic. Quality > quantity but even so, we're not judging applicants on raw technical abilities but instead their capacity and desire to learn.

It's not fun if master breaks but if it does, it's not the end of the world either. We have a manually-triggered "stable" deployment of the UI hosted at https://app.chrisproject.org which lags behind the automatic deployment of (working builds of) master branch on https://next.chrisproject.org

FYI the continuous delivery (CD) is achieved on Red Hat OpenShift, the documentation is here: https://github.com/FNNDSC/ChRIS_ui/wiki/Update-app.chrisproject.org
Prospective interns who are interested in CI/CD should reach out to me on Matrix.

**Review by jennydaman — CHANGES_REQUESTED** [review]
See comment below.

Please do not change code unrelated to the feature implementation, i.e. disable your IDE's automatic code formatter. [Discussed here, this is a violation of gitiquette](https://github.com/FNNDSC/ChRIS_ui/pull/676#issuecomment-1278298822) which can cause merge conflicts. I've created a new issue for this: https://github.com/FNNDSC/ChRIS_ui/issues/698

File: src/pages/VisualizationPage/index.tsx [hunk 1]
```diff
@@ -100,46 +105,57 @@ const VisualizationPage = () => {
 
   return (
     <Wrapper>
-      <div className="upload-link">
+      <PageSection variant="light">
+        <InfoIcon
+          title="DICOM Viewer"
+          p1={
+            <Paragraph style={infoStyle}>
+              Here you get to upload all your files.
```
Comment (jennydaman) [inline src/pages/VisualizationPage/index.tsx:113]: screenshot

Please update the tooltip. @PintoGideon @rudolphpienaar what do you think we should say here? Maybe...

**Review by jennydaman — CHANGES_REQUESTED** [review]
Oops, forgot to hit enter!

File: src/pages/VisualizationPage/index.tsx (hunk 1 above)
Comment (Timonwa) [reply src/pages/VisualizationPage/index.tsx]: I have updated it with the current suggestion.

**Comment by Timonwa** [comment]
> @Timonwa we actually did this before, take a look at what branches we have in this repository: https://github.com/FNNDSC/ChRIS_ui/branches
> 
> In previous runs of Outreachy, we asked contributors to target PRs to `outreachy/*` development branches. It led to a hell of merge conflicts because we were not strict about _gitiquette_. In the end we had to use cherry-picks and also leave out some of the contributions.
> 
> Currently, my intention is to be more strict about merging PRs and to have them target master. Hopefully this means we will be able to see more of people's work on our live deployment(s). If this gets out of hand, then we should think about having applicants target a new `outreachy/winter2022` branch.
> 
> p.s. **Unmerged PRs will still count towards your final Outreachy application.** Or more accurately, the Outreachy intern selection will be holistic. Quality > quantity but even so, we're not judging applicants on raw technical abilities but instead their capacity and desire to learn.
> 
> It's not fun if master breaks but if it does, it's not the end of the world either. We have a manually-triggered "stable" deployment of the UI hosted at https://app.chrisproject.org which lags behind the automatic deployment of (working builds of) master branch on https://next.chrisproject.org
> 
> FYI the continuous delivery (CD) is achieved on Red Hat OpenShift, the documentation is here: https://github.com/FNNDSC/ChRIS_ui/wiki/Update-app.chrisproject.org Prospective interns who are interested in CI/CD should reach out to me on Matrix.

Good to know that the deployment isn't automatic. I am going to read more on this ` Red Hat OpenShift`. Thank you for the reply.

**Comment by Timonwa** [comment]
Changed the DICOM Viewer hinnt message.
FireShot Capture 052 - DICOM Viewer - localhost

**Review by jennydaman — CHANGES_REQUESTED** [review]

File: src/components/common/info/InfoIcon.tsx [hunk 2]
```diff
@@ -28,20 +28,18 @@ const InfoIcon = ({
...
-              width: '0.75em',
-              marginLeft: '0.25em',
-            }}
-          />
-        </Popover>
-      </Title>
-    </div>
+    <Title>
+      {title}
+      <Popover placement='ottom'trigger='over'content={content}>
```
[the remark this answers is not in the document]
Comment (Timonwa) [reply src/components/common/info/InfoIcon.tsx:33]: Let me check it again

[the remark this answers is not in the document]
Comment (Timonwa) [reply src/components/common/info/InfoIcon.tsx]: I have checked it and confirmed that I have no errors again.

**Review by jennydaman — APPROVED** [review]

**Comment by jennydaman** [comment]
Looks good!

**Pull request merged**